subprojects/testing-jvm/src/integTest/groovy/org/gradle/testing/testsuites/dependencies/TestSuitesKotlinDSLDependenciesIntegrationTest.groovy

```diff
@@ -743,17 +740,16 @@ class TestSuitesKotlinDSLDependenciesIntegrationTest extends AbstractIntegration
                 suites {
                     val test by getting(JvmTestSuite::class) {
                         dependencies {
-                            implementation($dependencyNotation) { (dep: ModuleDependency) ->
-                                exclude(mapOf("group" to "commons-collections", "module" to "collections"))
-                                exclude(group = "commons-collections", module = "collections")
+                            implementation($dependencyNotation) {
+                                exclude(mapOf("group" to "commons-collections", "module" to "commons-collections"))
+                                exclude(group = "commons-collections", module = "commons-collections")
                             }
                         }
                     }
                 }
             }
 
             tasks.register("checkConfiguration") {
-                dependsOn("test")
```

Comment:
Is this why the sources were removed below? Does the GroovyDSL need the same tweak?

Reply:
No, but I removed it as it's not needed. I have now done the same with GroovyDSL.